.travis.yml

```diff
@@ -13,6 +13,7 @@ env:
     - COMPOSE_FILE=./containers/docker-compose-testing.yml
   matrix:
     - TASK="rake test:all"
+    - TASK="rails test"
```

Comment:
I've added this line to see if it gets regular (unit, functional, integration) tests running again?